### PR TITLE
Support otel.exporter.jaeger.timeout

### PR DESCRIPTION
**Pull request by trask**
Also includes minor doc fixes.

### CODE REVIEW

**Review by anuraaga — APPROVED** [review]
Thanks!

**Comment by jkwatson** [comment]
I don't see this in the specs. Should it be before we implement it? To make sure we don't have to end up changing the env var name after it's in the wild?

**Comment by trask** [comment]
😱 good catch, I didn't notice that. I will send spec PR.

**Comment by trask** [comment]
https://github.com/open-telemetry/opentelemetry-specification/pull/1612

File: sdk-extensions/autoconfigure/README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ The [Jaeger](https://www.jaegertracing.io/docs/1.21/apis/#protobuf-via-grpc-stab
 |-----------------------------------|-----------------------------------|----------------------------------------------------------------------------------------------------|
 | otel.traces.exporter=jaeger       | OTEL_TRACES_EXPORTER=jaeger       | Select the Jaeger exporter                                                                         |
 | otel.exporter.jaeger.endpoint     | OTEL_EXPORTER_JAEGER_ENDPOINT     | The Jaeger gRPC endpoint to connect to. Default is `http://localhost:14250`.                       |
+| otel.exporter.jaeger.timeout      | OTEL_EXPORTER_JAEGER_TIMEOUT      | The maximum waiting time allowed to send each batch. Default is `10000`.                           |
```
Comment (kubawach) [inline sdk-extensions/autoconfigure/README.md:68]: Perhaps adding unit (millis) here (and for OTLP readme) would be helpful for end-users (even though it's a common unit in similar contexts).

**Review by kubawach — APPROVED** [review]

**Comment by trask** [comment]
Spec PR merged 🥳

File: sdk-extensions/autoconfigure/README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ The [OpenTelemetry Protocol (OTLP)](https://github.com/open-telemetry/openteleme
 | otel.exporter.otlp.traces.endpoint  | OTEL_EXPORTER_OTLP_TRACES_ENDPOINT | The OTLP traces endpoint to connect to. Must be a URL with a scheme of either `http` or `https` based on the use of TLS. Default is `http://localhost:4317`.            |
 | otel.exporter.otlp.metrics.endpoint  | OTEL_EXPORTER_OTLP_METRICS_ENDPOINT | The OTLP metrics endpoint to connect to. Must be a URL with a scheme of either `http` or `https` based on the use of TLS. Default is `http://localhost:4317`.            |
 | otel.exporter.otlp.headers   | OTEL_EXPORTER_OTLP_HEADERS  | Key-value pairs separated by commas to pass as request headers.        |
-| otel.exporter.otlp.timeout   | OTEL_EXPORTER_OTLP_TIMEOUT  | The maximum waiting time allowed to send each batch. Default is `1000`.   |
+| otel.exporter.otlp.timeout   | OTEL_EXPORTER_OTLP_TIMEOUT  | The maximum waiting time allowed to send each batch. Default is `10000`.  |
```
Comment (jkwatson) [inline sdk-extensions/autoconfigure/README.md:55]:
```suggestion
| otel.exporter.otlp.timeout   | OTEL_EXPORTER_OTLP_TIMEOUT  | The maximum waiting time, in milliseconds, allowed to send each batch. Default is `10000`.  |
```

**Review by jkwatson — APPROVED** [review]

**Pull request merged**